src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Blank.txt

```diff
@@ -91,7 +91,6 @@ true
 0
 
 >> Blank() = Blank()
-/*InterpreterRunner*/ #Skip
```

Comment:
Engine-specific issues are not called out via a dedicate engine-specific .txt file in that engine's repo.  See BlankOverride.txt for example here.